### ISSUE TITLE
Develop an approach for settings navigation, changes, restoration, etc.

### ISSUE THREAD

**Issue by chris-bc** [chris-bc/Flipper-Gravity#13]
Once sync is in place some consideration needs to be given to how changes are made, saved and restored.
* When a config option is changed from the UI, app->selected_menu_options will be updated. Maintain this behaviour. -- store modified selections in temp location, reverting to previous if enter not pressed. Then no need for Revert
* Pressing enter on an option will save that option (back to esp32-gravity)
* Add a new option to revert settings, restoring them from sync()
* Display confirmation of saving using a modal dialogue

The above doesn't feel quite right. Get this right and then build it. Also create a new issue to save and load named configurations to and from Flipper

**Comment by chris-bc** [comment]
Item (2) in place, however also needs to update Flipper menu item(s)
